packages/material-ui/src/Breadcrumbs/BreadcrumbCollapsed.js

```diff
@@ -32,10 +33,11 @@ const styles = theme => ({
  */
 function BreadcrumbCollapsed(props) {
   const { classes, ...other } = props;
+
```

Comment:
@oliviertassinari Without 
```javascript
const handleKeyDown = e => e.preventDefault
```
[Here](https://material-ui.com/components/breadcrumbs/#collapsed-breadcrumbs), if you click `Space`, page will scroll down.
Do I need to update example, to prevent scrolling down ?

Reply:
The focus is trapped with this logic, plus the button base should already handle it.

Reply:
Diving into it, hold on.

Reply:
The `li > button > svg` structure would solve this problem but it sounds like a regression.

Reply:
Ok, it seems to be a bug in the ButtonBase, I believe the fix is:

```diff
diff --git a/packages/material-ui/src/ButtonBase/ButtonBase.js b/packages/material-ui/src/ButtonBase/ButtonBase.js
index f34c70028..7fc1be633 100644
--- a/packages/material-ui/src/ButtonBase/ButtonBase.js
+++ b/packages/material-ui/src/ButtonBase/ButtonBase.js
@@ -210,6 +210,10 @@ const ButtonBase = React.forwardRef(function ButtonBase(props, ref) {
       onKeyDown(event);
     }

+    if (event.target === event.currentTarget && isNonNativeButton() && event.key === ' ') {
+      event.preventDefault();
+    }
+
     // Keyboard accessibility for non interactive elements
     if (event.target === event.currentTarget && isNonNativeButton() && event.key === 'Enter') {
       event.preventDefault();
@@ -240,15 +244,13 @@ const ButtonBase = React.forwardRef(function ButtonBase(props, ref) {

     // Keyboard accessibility for non interactive elements
     if (
+      onClick &&
       event.target === event.currentTarget &&
       isNonNativeButton() &&
       event.key === ' ' &&
       !event.defaultPrevented
     ) {
-      event.preventDefault();
-      if (onClick) {
-        onClick(event);
-      }
+      onClick(event);
     }
   });

diff --git a/packages/material-ui/src/ButtonBase/ButtonBase.test.js b/packages/material-ui/src/ButtonBase/ButtonBase.test.js
index cc7e2e102..adfa2ff6d 100644
--- a/packages/material-ui/src/ButtonBase/ButtonBase.test.js
+++ b/packages/material-ui/src/ButtonBase/ButtonBase.test.js
@@ -712,7 +712,7 @@ describe('<ButtonBase />', () => {

         expect(onClickSpy.callCount).to.equal(1);
         // defaultPrevented?
-        expect(onClickSpy.returnValues[0]).to.equal(true);
+        expect(onClickSpy.returnValues[0]).to.equal(false);
       });

       it('does not call onClick when a spacebar is released and the default is prevented', () => {
```

Thanks for the report. If you have time for a follow-up pull request, it could be a great topic to cover.

Reply:
@oliviertassinari I will fix it in scope of this PR

Reply:
Hum, I would recommend a different pull request, the scope of the impact is way different.

Reply:
Ok, I will create an issue today with PR